index.html

```diff
@@ -1,227 +1,326 @@
+                    </div>
+                </div>
+              </li>
+            <li class="p-matrix__item">
+              <div class="p-matrix__content">
+                <h3 class="p-matrix__title">Upstream</h3>
+                  <div class="p-matrix__desc">
+                    <p>CNCF binaries delivered to your laptop, with updates and upgrades.</p>
+                  </div>
+              </div>
+            </li>
+            <li class="p-matrix__item">
+                <div class="p-matrix__content">
+                  <h3 class="p-matrix__title">Complete</h3>
+                    <div class="p-matrix__desc">
+                      <p>Includes a docker registry so you can make containers, push them, and deploy them all on your laptop.</p>
+                    </div>
+                </div>
+            </li>
+            <li class="p-matrix__item">
+                  <div class="p-matrix__content">
+                    <h3 class="p-matrix__title">Featureful</h3>
+                      <div class="p-matrix__desc">
+                        <p>Cool things you probably want to try on a small, standard K8s are all built-in. Just enable them and go.</p>
+                      </div>
+                  </div>
+            </li>
+            <li class="p-matrix__item">
+              <div class="p-matrix__content">
+                <h3 class="p-matrix__title">Updates</h3>
+                  <div class="p-matrix__desc">
+                    <p>Get the daily build if you want it, or betas and milestones, or just stable post releases.</p>
+                  </div>
+              </div>
+            </li>
+            <li class="p-matrix__item">
+              <div class="p-matrix__content">
+                <h3 class="p-matrix__title">Upgrades</h3>
+                  <div class="p-matrix__desc">
+                    <p>When a new major version comes out, upgrade with a single command (or automatically).</p>
+                  </div>
+              </div>
+            </li>
+            <li class="p-matrix__item">
+              <div class="p-matrix__content">
+                <h3 class="p-matrix__title">GPGPU Passthrough</h3>
+                  <div class="p-matrix__desc">
+                    <p>Give microk8s a GPGPU and your docker containers can get all nice and CUDA.</p>
+                  </div>
+                </div>
+            </li>
+            <li class="p-matrix__item">
+                <div class="p-matrix__content">
+                  <h3 class="p-matrix__title"> </h3>
+                    <div class="p-matrix__desc">
+                      <p> </p>
+                    </div>
+                  </div>
+              </li>                                                                           
+          </ul>
         </div>
-      </div>
-      <div class="row mt centered mobile-only">
-        <div class="col-lg-12">
+      </section>
+
+      <section><a name="quick-start"></a>
```

Comment:
I think it's better to put an ID on the `section` element here rather than an empty link